apps/meteor/app/apps/client/orchestrator.ts

```diff
@@ -89,7 +89,7 @@ class AppClientOrchestrator {
 	}
 
 	public async getApps(): Promise<App[]> {
-		const result = await APIClient.get<'/apps'>('/apps');
+		const result = await APIClient.get<'/apps/installed'>('/apps/installed');
```

Comment:
IMO `getApps` should get **all** apps, not only installed ones - semantically speaking. Is it supposed to fetch just the installed ones? Maybe rename the method?

Reply:
It doesn´t necessarily returns installed apps but the only purpose of the return of this endpoint/function is to become the list of installed apps. So, I'll change the title of the function. Solved.